codegen/smithy-ruby-codegen-test/model/component-test/compression.smithy

```diff
@@ -0,0 +1,35 @@
+$version: "1.0"
+namespace smithy.ruby.tests
+
+@requestCompression(
+    encodings: ["gzip"]
+)
+
+@http(method: "POST", uri: "/some_operation")
+operation SomeOperation {
```

Comment:
Rather than calling this SomeOperation we should make it a bit more specific/descriptive. Maybe requestCompressuonOperation or something - since there are a lot of operations in this model testing different features.